R/benchmark.R

```diff
@@ -36,3 +36,33 @@ run_plant_benchmarks <- function(strategy_types = list(FF16 = FF16_Strategy,
                  )
                })
 }
+
```

Comment:
Can we add a comment here to explain what this function is doing? Running some sort of benchmark, but what is it benchmarking

Reply:
How about the comment below? I think this was originally used to compare two different implementations of the consumption framework. If this is no longer relevant to track we can remove it.